clippy_dev/src/new_lint.rs

```diff
@@ -63,6 +63,14 @@ pub fn create(
         add_lint(&lint, msrv).context("Unable to add lint to clippy_lints/src/lib.rs")?;
     }
 
+    if pass == "early" {
+        println!(
+            "\n\
+            NOTE: Use a late pass unless you need something specific from\n\
+            an early pass, as they lack many features and utilities"
```

Comment:
You convinced me that this is a good idea. But I would remove the `\n` from the message and let the user terminal insert the line ~~breaks~~wraps. Otherwise this will look really weird, if the terminal inserts extra line wraps